monai/transforms/intensity/array.py

```diff
@@ -1132,7 +1132,7 @@ class ScaleIntensityRangePercentiles(Transform):
     .. code-block:: python
         :emphasize-lines: 11, 22
 
-        image = np.array(
+        image = torch.Tensor(
```

Comment:
The type of the `image` may not be the key point, the input dtype is the reason for the error message. It's possible that torch and numpy produce different default dtype.

Reply:
Hi @KumoLiu! Yes definitely, if I use `np.array` with `dtype=np.float32`, it works. Numpy choses np.int64 by default for this list. Nevertheless the return type is now a metatensor, so the prints are incorrect anyways.

Reply:
Yes, the update should be fine and just wanted to bring that up.

Reply:
But you are correct that this is not a bug then, it's just the wrong input dtype :) Just checked, typing is fine too, not sure what I saw there..